src/aws_lc_rs_algs.rs

```diff
@@ -39,6 +45,19 @@ impl SignatureVerificationAlgorithm for AwsLcRsAlgorithm {
     }
 }
 
+fn ec_compressed_or_uncompressed(public_key: &[u8]) -> Result<(), InvalidSignature> {
+    // "The first octet of the OCTET STRING indicates whether the key is
+    //  compressed or uncompressed.  The uncompressed form is indicated
+    //  by 0x04 and the compressed form is indicated by either 0x02 or
+    //  0x03 (see 2.3.3 in [SEC1]).  The public key MUST be rejected if
+    //  any other value is included in the first octet."
+
+    match public_key.first() {
```

Comment:
nit: the whitespace on L54 seems unnecessary. Also I think `[SEC1]` is missing a target URL (and wouldn't be rendered as markdown since this is a `//` comment internal to the fn. 

Maybe best to promote this to an internal rustdoc comment on the fn and add the SEC1 URL?

Reply:
Put this in a rustdoc, and added a link to the section of RFC5280 I'm quoting. (haven't done anything with the dangling SEC1 reference, since I think it's better to leave this as a straight pull quote.)

Reply:
>  better to leave this as a straight pull quote

Oh, that makes sense. I overlooked this was text/markup straight from the source. :+1:

Reply:
OK, scratch that. Now rustdoc is complaining. So I've escaped the reference.